packages/core-svelte/src/components/TextDropdownComponent.svelte

```diff
@@ -325,8 +322,8 @@
         }
     };
 
-    const onFocusOut = () => {
-        LOGGER.log("TextDropdownComponent.onFocusOut");
+    const onBlur = () => {
```

Comment:
Does this mean that the todos are resolved? Please remove them - in that case.

Reply:
resolved todo's